### ISSUE TITLE
Unable to retrieve Drupal project from ftp...

### ISSUE THREAD

**Issue by dminca** [dminca/drupal-docker#45]
After hitting the Install button in the Admin -> Extend menu to install a Theme or Module, the following error is thrown:

``` 
Failed to fetch file due to error "Error creating resource: [message] fopen(): SSL operation failed with code 1. OpenSSL Error messages: error:14090086:SSL routines:ssl3_get_server_certificate:certificate verify failed [file] /var/www/html/vendor/guzzlehttp/guzzle/src/Handler/StreamHandler.php [line] 282 [message] fopen(): Failed to enable crypto [file] /var/www/html/vendor/guzzlehttp/guzzle/src/Handler/StreamHandler.php [line] 282 [message] fopen(https://ftp.drupal.org/files/projects/devel-8.x-1.x-dev.tar.gz): failed to open stream: operation failed [file] /var/www/html/vendor/guzzlehttp/guzzle/src/Handler/StreamHandler.php [line] 282"
    Unable to retrieve Drupal project from https://ftp.drupal.org/files/projects/devel-8.x-1.x-dev.tar.gz.
```

### Useful resources
* [How to install ssh2 php7 extension][1]

### Suggestions
- [x] **[FAILED]** try and start the `completit/dockerized-drupal:ubuntu` container, and compare this `phpinfo()` with the `completit/dockerized-drupal:alpine` container for missing PHP modules
- [x] **[FAILED]** check whether `php7-ftp` is [enabled][2] on `completit/dockerized-drupal:alpine` container ( `'--enable-ftp'` )
- [x] **[FAILED]** try the [bruteforce][4] way `chmod 777 sites modules` from the  `/app` folder
- [x] **[PASSED]** try another [suggestion][3] `sudo chown -R www-data:www-data app` (`www-data` is for apache, `nginx` is for nginx)

**Relates to** #44 

[1]: https://medium.com/php-7-tutorial/solution-how-to-compile-php7-with-ssh2-f23de4e9c319#.dfjalpa1t
[2]: https://www.drupal.org/node/1024032#comment-4181620
[3]: https://www.drupal.org/node/1024032#comment-5490522
[4]: https://www.drupal.org/node/2042447#comment-10194583

**Comment by dminca** [comment]
**OpenSSL cafile** is not available on **PHP 7.0.6** `completit/dockerized-drupal:ubuntu` image
![selection_004](https://cloud.githubusercontent.com/assets/5355219/16269636/d2d6cebc-389b-11e6-9681-cdc59b222725.png)

**Comment by dminca** [comment]
Issue still present in the `completit/dockerized-drupal:alpine` image
